core/src/test/java/io/aiven/klaw/TopicRequestValidatorImplIT.java

```diff
@@ -361,17 +359,17 @@ public void isValidTestVerifyIfTopicAlreadyExists() {
         .contains("Failure. This topic already exists in the selected cluster.");
   }
 
-  @Test
```

Comment:
is it possible to add similar tests (which are removed) in TopicControllerService or somewhere to test the permission annotations?

Reply:
Added tests